src/gate/index.js

```diff
@@ -54,21 +57,46 @@ Object.assign(UserGate.prototype, {
     return this._matchesSample(user) || this._matchesList(user);
   },
 
+  allowsUniform: function(user) {
```

Comment:
Can we add docs on this? Or update the docs on `allows` per my comment about merging into that.

Reply:
> update the docs on allows per my comment about merging into that.

https://github.com/mixmaxhq/user-gate/pull/5/commits/7e6dfa937363205c20409112b4cd4f9e5553baf0